Src/BootCamp.Chapter/Program.cs

```diff
@@ -1,14 +1,75 @@
 ﻿using System;
-using System.Collections.Generic;
-using System.IO;
+using static System.Math;
 using Console = System.Console;
 
 namespace BootCamp.Chapter
 {
     class Program
     {
+        /*  Homework 3
```

Comment:
Move this comment as an xml summary above either Lesson3 class or Demo method. Like that, hovering your mouse over that method will show its' description. You don't want to force it in code, it messes up the readability.

Reply:
Done